### PR TITLE
Moved content of 'shared' folder into MusicStore app and fixed tests

### PR DESCRIPTION
**Pull request by kichalla**
@pranavkm These are fixes related to make MusicStore repo work for changes https://github.com/aspnet/Hosting/pull/828

### CODE REVIEW

**Comment by kichalla** [comment]
:up: :date:

File: test/E2ETests/MusicStoreConfig.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
```
Comment (pranavkm) [inline test/E2ETests/MusicStoreConfig.cs]: Remove usings.

File: src/MusicStore/ForTesting/Mocks/OpenIdConnect/OpenIdConnectBackChannelHttpHandler.cs [hunk 2]
```diff
@@ -13,8 +12,7 @@ protected override Task<HttpResponseMessage> SendAsync(HttpRequestMessage reques
             var response = new HttpResponseMessage();
 
             var basePath = Path.GetFullPath(Path.Combine(
-                    Directory.GetCurrentDirectory(), "..", "..",
-                    "shared", "Mocks","OpenIdConnect"));
+                Directory.GetCurrentDirectory(), "ForTesting", "Mocks","OpenIdConnect"));
```
Comment (pranavkm) [inline src/MusicStore/ForTesting/Mocks/OpenIdConnect/OpenIdConnectBackChannelHttpHandler.cs]: Space after `"Mocks",`

File: src/MusicStore/ForTesting/Mocks/Common/Helpers.cs [hunk 3]
```diff
@@ -1,5 +1,4 @@
-﻿#if TESTING
-using System;
+﻿using System;
```
Comment (pranavkm) [inline src/MusicStore/ForTesting/Mocks/Common/Helpers.cs]: Could we follow up to remove this type?

**Comment by pranavkm** [comment]
:shipit:

File: src/MusicStore/ForTesting/Mocks/Common/Helpers.cs (hunk 3 above)
Comment (kichalla) [inline src/MusicStore/ForTesting/Mocks/Common/Helpers.cs]: Sure

**Pull request merged**